data/autoyast_hanaperf/sle15.xml.ep

```diff
@@ -32,10 +32,10 @@
       <listentry>
         <alias>SLE-Module-Basesystem</alias>
         % if ($check_var->('VERSION', '15-SP1')) {
```

Comment:
why not to simplify all this section by removing all the `if/else`  and have `media_url` directly from a Job setting?

Reply:
Thank you, let me make this simpler because we will not cover 15-SP1 already.

Reply:
Sorry, because we are shareing openqa at Beijing, we uses this method to override original job setting (pointing to mirror.suse.asia)